### ISSUE TITLE
Ethanol high in blood

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10034]
**Preferred term label:**
Ethanol high in blood

**Synonyms**
Alcohol intoxication

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C2711518

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Auto-brewery syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
What definition should we use for "high"?

**Comment by MickeySegal** [comment]
There are a variety of levels and consequences at https://www.urmc.rochester.edu/encyclopedia/content.aspx?contenttypeid=167&contentid=ethanol_blood

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7667719/ describes "signs of alcohol intoxication" but uses a lower limit of 0.0005 g/dL, but on the scale above that is almost sober, while alcohol intoxication would require at least 0.0800 g/dL.

So I'd favor either 0.08 or 0.10 g/dL/

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000986
- label: Substantially elevated circulating ethanol concentration
- parents: [Blood xenobiotic (HP:0410172)](https://hpo.jax.org/app/browse/term/HP:0410172)
- pmid: PMID:33209539
- def: A blood concentration of ethanol that is consistent with alcohol intoxication, generally assumed to be 0.0800 g/dL or higher.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10034
- synonyms:
